task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -192,7 +203,7 @@ def exit(n: int) -> NoReturn:
             sys.stdout.flush()
         with suppress(ValueError, OSError):
             sys.stderr.flush()
-        with suppress(ValueError, OSError):
+        with suppress(ValueError, OSError, AttributeError):
```

Comment:
What could cause an `AttributeError` here?

Reply:
pytest

Reply:
YEah not needed anymore. Will remove